concepts/Common/Data Analytics and BI/Embedded Reports/Embedded Reports.md

```diff
@@ -0,0 +1,17 @@
+Explore [DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/) - a reporting solution designed to meet your needs for advanced reporting functionality, rich design-time experience, and extensive customization options.
+
+![DevExpress Reporting](/images/Common/Embedded_Reports.png)
+
+#### Key Features
+
+- Subreports, master-detail reports, report parameters, expressions, data bindings, and conditional formatting.
+
+- An intuitive drag-and-drop interface and [integration with Visual Studio](https://docs.devexpress.com/XtraReports/404722/vs-code-report-designer).
```

Comment:
Let's add VS code also:
An intuitive drag-and-drop interface and integration with [Visual Studio](https://docs.devexpress.com/XtraReports/404722/vs-code-report-designer) and VS Code (link).